postprocessing/visualization/tools/make_animation.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+import argparse
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib import animation
+from seissolxdmf import seissolxdmf as sx
+
+plt.rcParams['savefig.facecolor'] = 'w'
+
+
+def main():
+    description = '''Creates an animation of the velocity/displacement
+    wavefield from SeisSol free-surface XDMF output file.'''
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument(
+        'fname', help='SeisSol surface (XDMF) output file.')
+    parser.add_argument(
+        'dataid', help='Which data component to use', default='v1')
+    parser.add_argument(
+        '--cmap', help='Matplotlib colormap', default='bwr')
+    parser.add_argument(
+        '--dpi', default=200, type=int)
+    parser.add_argument(
+        '--bounds', nargs='+', help='xmin xmax ymin ymax', type=float)
+    args = parser.parse_args()
+
+    ds = sx(args.fname)
+    data = ds.ReadData(args.dataid)
+    geo = ds.ReadGeometry()
+    connect = ds.ReadConnect()
+    dt = ds.ReadTimeStep()
+    nt = data.shape[0]
+    tarr = np.arange(0.0, nt * dt, dt)
+
+    fig = plt.figure(figsize=(8, 7))
+    ax = plt.subplot(111)
+    dmax = abs(data).max()
+    im = ax.tripcolor(
+        geo.T[0], geo.T[1], connect, data[0], cmap=args.cmap, vmin=-dmax,
+        vmax=dmax)
+    ax.set_xlabel('Easting (m)')
+    ax.set_ylabel('Northing (m)')
```

Comment:
This only holds for ENU meshes. I'd prefer {x, y} here, since some user might use a different coordinate system.